app/Assessment.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use App\Section;
+use App\AssessmentItem;
+
+class Assessment extends Model
+{
+    public function section() {
+        return $this->belongsTo('Section');
```

Comment:
`$this->belongsTo(Section::class)` 

do this on all models which allows you to jump to the file in the IDE and also highlight it if the other class does not exist.